sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/OrderByUtils.java

```diff
@@ -49,7 +49,8 @@ public static <T extends Resource> Flux<OrderByRowResult<Document>> orderedMerge
                                                        targetRangeToOrderByContinuationTokenMap,
                                                        consumeComparer.getSortOrders(), clientSideRequestStatistics))
                 .toArray(Flux[]::new);
-        return Flux.mergeOrdered(consumeComparer, fluxes);
+        // prefetch is set to 1 to minimize the no. prefetched pages per partition
+        return Flux.mergeOrdered(1, consumeComparer, fluxes);
```

Comment:
Flux.mergeOrdered is going to be deprecated, maybe we can change to use the latest `mergeComparingDelayError`

Reply:
Made the change.